src/anomalib/models/image/vlm_ad/utils.py

```diff
@@ -0,0 +1,25 @@
+"""Dataclasses."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import dataclass
+from enum import Enum
+
+
+@dataclass
+class Prompt:
+    """Prompt."""
+
+    few_shot: str
+    predict: str
+
+
+class VLMModel(Enum):
```

Comment:
so, `VLMModel` is the name of the models, and `Backend` returns the model implementation? Did I get it correctly?

Reply:
Yeah. The backend is chosen based on the model name.

Reply:
I'm thinking for future, would we ever need `VLMModel` name for an actual model, instead of an enum